<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountInfoPage</class>
 <widget class="QDialog" name="AccountInfoPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#AccountInfoPage{
background-color: #eaf2f8;
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>200</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#label {
        color: #2c3e50;
        font-size: 20px;
        font-weight: bold;
    }</string>
   </property>
   <property name="text">
    <string>اطلاعات حساب</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>540</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> #pushButton {
        background-color: #2980b9;
        color: white;
        border: none;
        border-radius: 6px;
        padding: 8px 16px;
        font-size: 14px;
    }
    #pushButton:hover {
        background-color: #3498db;
    }
    #pushButton:pressed {
        background-color: #1c5980;
    }</string>
   </property>
   <property name="text">
    <string>تایید</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>621</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame {
        background-color: qlineargradient(
            spread:pad, x1:0, y1:0, x2:1, y2:1,
            stop:0 #34495e, stop:1 #2c3e50
        );
        color: white;
        border-radius: 16px;
        padding: 12px;
        border: 1px solid #1c2833;
    }</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="accountType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#accountType {
	color: rgb(255, 255, 255);
        font-size: 16px;
        font-weight: bold;
    }</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="shebaNumber">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#shebaNumber {
	color: rgb(255, 255, 255);
        font-size: 16px;
        font-weight: bold;
    }</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cardNumber">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>160</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cardNumber {
	color: rgb(255, 255, 255);
        font-size: 16px;
        font-weight: bold;
    }</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="owner">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>400</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#owner {
	color: rgb(255, 255, 255);
        font-size: 16px;
        font-weight: bold;
    }</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cvv2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#cvv2 {
	color: rgb(255, 255, 255);
        font-size: 16px;
        font-weight: bold;
    }</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="expirationDate">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>300</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#expirationDate {
	color: rgb(255, 255, 255);
        font-size: 16px;
        font-weight: bold;
    }</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
